lib/tests/streamlit/connections/util_test.py

```diff
@@ -48,6 +47,14 @@ def test_not_running_in_sis(self):
     def test_running_in_sis(self):
         assert running_in_sis()
 
+    @pytest.mark.require_snowflake
```

Comment:
Having to do this is mildly unfortunate, but we can't mock `snowflake.*` without having `snowflake-connector-python` installed. In reality, having either the snowflake or snowpark libraries not installed will result in a `ModuleNotFoundError` being thrown.